3p/ampcontext.js

```diff
@@ -266,6 +266,33 @@ export class AbstractAmpContext {
     this.client_.sendMessage(MessageType.NO_CONTENT);
   }
 
+  /**
+   * Function to register a callback on the iframe messaging client.
+   * This is to accept messages of any type, for 3p communication.
+   * Please use the specific functions for context messaging where possible.
+   * @param {string} messageType
+   * @param {function(?JsonObject)} callback
+   */
+  registerCallback(messageType, callback) {
```

Comment:
I agree, should not break my tests. I'll just replace the messaging code in recaptcha.js with a TODO for now, since we are going to rewrite it anyways soon. 😄